packages/grafana-data/src/types/logs.ts

```diff
@@ -33,6 +33,11 @@ export enum LogsMetaKind {
   LabelsMap,
 }
 
+export enum LogsSortOrder {
+  Descending = 'Descending',
+  Ascending = 'Ascending',
+}
+
```

Comment:
moved from core to grafana/data so we can use it in LogRows that is in grafana/ui